control/init.sh

```diff
@@ -140,10 +141,13 @@ BLUE='\033[0;34m'
 NC='\033[0m' # No Color
 SITE_PACKAGES=$(python -c 'import site; print(site.getsitepackages()[0])')
 
-if ! grep -q init-ansible ~/.profile 2>/dev/null; then
-    printf "${YELLOW}Do you want to have the CommCare Cloud environment setup on login?${NC}\n"
-    if [ -z ${CI_TEST} ]; then
-        read -t 30 -p "(y/n): " yn
+if [ -z ${CI_TEST} ]; then
```

Comment:
I did want to flag this change. To me it seems if we are running this in the context of a test suite, we wouldn't care about any of the logic around automatically have the commcare cloud environment setup on login, not just the line that attempts to read user input. Just want to confirm that fits other reviewers' understandings as well.